draft-terbu-oauth-sd-jwt-vc.md

```diff
@@ -111,6 +111,8 @@ SD-JWTs VC can contain claims that are registered in "JSON Web Token Claims"
 registry as defined in [@!RFC7519], as well as public and
 private claims.
 
+Note: This specification does not utilize [@W3C.VCDM].
```

Comment:
```suggestion
Note: This specification does not utilize the W3C's Verifiable Credentials Data Model.

```

Reply:
and delete the whole `<reference anchor="W3C.VCDM / VC-DATA" target="https://www.w3.org/TR/vc-data-model-2.0/">...<reference/>` block (which is more than GH will let me suggest)

Reply:
```suggestion
Note: This specification does not utilize the W3C's Verifiable Credentials Data Model v1.0, v1.1, or v2.0.
```